PNG rendering fails with graphviz on Windows

When I render the following block:

```
[graphviz,"test",svg]
----
graph G {
  clusterC -- clusterB
}
----
```

I get the following error message: `(RuntimeError) Invalid PNG signature`

I am using the basic maven setup from the maven examples (asciidoctor-diagram-example) on a french Windows 7 Pro.

I patched the file 'png.rb' which causes this error by returning a size by default. With this patched version the png file gets created but is corrupted. 

I suspect this being an encoding issue UTF-8 vs ISO.

(Btw, I also tried generating the png on my command line with the exact same parameters and it works.)

Below the complete error output.

```
[INFO] --- asciidoctor-maven-plugin:1.5.0:process-asciidoc (generate-html-doc) @
 asciidoctor-diagram-example ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.248 s
[INFO] Finished at: 2014-11-06T13:30:58+01:00
[INFO] Final Memory: 12M/30M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:1.5.0:pr
ocess-asciidoc (generate-html-doc) on project asciidoctor-diagram-example: Execu
tion generate-html-doc of goal org.asciidoctor:asciidoctor-maven-plugin:1.5.0:pr
ocess-asciidoc failed: (RuntimeError) Invalid PNG signature -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e swit
ch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please rea
d the following articles:
```

--- a/lib/asciidoctor-diagram/util/cli_generator.rb
+++ b/lib/asciidoctor-diagram/util/cli_generator.rb
@@ -31,7 +31,7 @@ module Asciidoctor
 
           raise "#{tool} image generation failed" unless result_code == 0
 
-          File.read(target_file.path)
+          File.read(target_file.path, :mode => 'rb')
         ensure
           target_file.unlink
         end

